tests/integration/test_fim/conftest.py

```diff
@@ -0,0 +1,131 @@
+import os
```

Comment:
Missing header. Check all files with this problem.

Reply:
Done